pai/model.py

```diff
@@ -987,12 +987,18 @@ def _deploy_local(
 
         # build command to install requirements
         if requirements_list:
-            install_requirements = shlex.join(
-                ["python", "-m", "pip", "install"] + requirements_list
+            install_requirements = " ".join(
+                [
+                    shlex.quote(s)
+                    for s in ["python", "-m", "pip", "install"] + requirements_list
```

Comment:
suggestion: for s in (["python", "-m", "pip", "install"] + requirements_list)

Reply:
I had considered it before and thought it was a piece of redundant code.